roles/openshift_aws/tasks/iam_role.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Create an iam role
+  iam_role:
+    name: "{{ item.value.iam_role }}"
+    assume_role_policy_document: "{{ lookup('file','trustpolicy.json') }}"
+    state: "{{ openshift_aws_iam_role_state | default('present') }}"
+  with_dict: "{{ l_nodes_to_build }}"
+
+- name: create an iam policy
```

Comment:
We should add some comments here to the what/why part of doing this IAM role.  It's not really clear why this is being done.

Reply:
Updated with some explanation.